cypress/e2e/app.cy.ts

```diff
@@ -0,0 +1,43 @@
+import { tabs, TOKEN_LIST } from '../support/constants';
+
+describe('App interaction', () => {
+  beforeEach(() => {
+    // cy.interceptAllResquests();
+  });
```

Comment:
Doesn't seem to be used this beforeEach, remove?

Reply:
https://github.com/0xShishigami/feo-web3-react-boilerplate/pull/6/commits/7edf124f7352a4680f0807e8c6f4d61278ab5035